Copy edit 5.16 guides (#2010)
sensu/sensu-docs: content/sensu-go/5.16/guides/reduce-alert-fatigue.md

  ttl: 0
{{< /highlight >}}

Notice the annotations under the `metadata` scope.
The annotations are required for the filter asset to work the same way as the interactively created event filter.
Specifically, the annotations in this check definition are doing several things: 

1. `fatigue_check/occurrences`: Tells the event filter on which occurrence to send the event for further processing
2. `fatigue_check/interval`: Tells the event filter the interval at which to allow additional events to be processed (in seconds)

> 3. `fatigue_check/allow_resolution`: Determines wiether to pass a `resolve` event through to the filter

Reviewer: ```suggestion
3. `fatigue_check/allow_resolution`: Determines whether to pass a `resolve` event through to the filter
```

> For more information about configuring these values, see the [filter asset README][10].

Reviewer: ```suggestion
For more information about configuring these values, see the [filter asset's README][10].
```

Next, you'll assign the newly minted event filter to a handler.

#### Assign the event filter to a handler

> Just like with the [interactively created event filter][4], you'll assign the filter to a handler.

Reviewer: ```suggestion
Just like with the [interactively created event filter][4], you'll introduce the filter into your Sensu workflow by configuring a handler to use it.
```
